Source/DafnyTestGeneration/ProgramModifier.cs

```diff
@@ -268,7 +268,7 @@ private class AnnotationVisitor : StandardVisitor {
         node.Blocks[0].cmds.Insert(0, GetAssumeCmd(types));
 
         // record parameter values:
-        var values = new List<string> { "\"Impl\"", $"\"{implName}\"" };
+        var values = new List<string> { "\"Impl\"", $"\"{node.VerboseName.Split(" ")[0]}\"" };
```

Comment:
I'm slightly worried that this would be fragile if we change the text in `VerboseName` at any point. Of all the string matching you could do, though, taking the first "word" is probably the safest, so there's probably no need to change this for now.

Reply:
Having VerboseName split into two fields of which one identifies the Dafny name and the other - the purpose would make this cleaner but as it stands right now, this was the best I could come up with, unfortunately.